riscv/csrs.cc

```diff
@@ -1561,6 +1561,18 @@ void senvcfg_csr_t::verify_permissions(insn_t insn, bool write) const {
   masked_csr_t::verify_permissions(insn, write);
 }
 
+// senvcfg.sse is read_only 0 when menvcfg.sse = 0
+reg_t senvcfg_csr_t::read() const noexcept {
+  reg_t envcfg = (state->v) ? state->henvcfg->read() : state->menvcfg->read();
```

Comment:
@binno - one nit - (state->v)  can be just state->v. The parenthesis are not needed.